src/components/Header/Header.tsx

```diff
@@ -9,11 +9,161 @@ const navItems: INavItem[] = [
     id: uid(),
     name: 'Products',
     link: '/products',
+    content: [
```

Comment:
Please let's move this list of items to different files and import them here. it makes the file very huge. You can just create a file alongside each of these files.